Src/VimWpf/VimHost.cs

```diff
@@ -219,6 +219,15 @@ public virtual bool IsDirty(ITextBuffer textBuffer)
             return false;
         }
 
+        public virtual bool IsLoaded(ITextView textView)
+        {
+            if (textView is IWpfTextView wpfTextView)
+            {
+                return wpfTextView.VisualElement.IsLoaded;

```

Comment:
Curious: why does the trick in `SetupInitialMode`, where we just check to see if we can get the lines, not work for this change?

Reply:
In my debugging, the `GetTextViewLines` became available, but it did not reflect the final state of the text view. For example, our viewport is (say) fifty lines high, but the text view lines collection only had length two. The method `SetupInitialMode` only cares that Visual Studio has properly initialized the view settings (like tabstop and shiftwidth); it doesn't care about the actual view itself. By contrast, when we are ensuring view properties like "is visible" or "scroll offset", we really need an accurate picture of text view in relation to the viewport.

Prior to WPF transitioning the element to `IsLoaded <- true`, the VisualElement is not even part of the visual tree yet. It has no parent element, blah, blah, blah, and so it couldn't possibly be sized correctly or answer our critical viewport related questions or operations properly yet.